lib/lsperfm/core/run.rb

```diff
@@ -18,9 +18,9 @@ class Runner
 
     attr_reader :command
 
-    def initialize(config, debug = false, logstash_home = Dir.pwd)
+    def initialize(config, workers = "1", debug = false, logstash_home = Dir.pwd)
```

Comment:
Yeah, the reason workers is a string here is because popen expects a string and fails if it's an integer. I suppose we could have it set to an int here and then call .to_s when setting the variable that gets sent to popen